### PR TITLE
[HANDL-509] correct some representational errors, e.g. number to integer and other wrong formats

### PR DESCRIPTION
**Pull request by tibortezsla**
## Problem

There were some problems with the schemas. E.g. some were `string` instead of `boolean`, some were `number` instead of `integer` 

## Proposed changes

Corrected the above mistakes. Tested it, and all the streams work correctly.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Unit tests for changes (not needed for documentation changes)
- [ ] CI checks pass with my changes
- [ ] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [x] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [x] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: tap_twilio/schemas/cumulative_statistics.json [hunk 1]
```diff
@@ -79,91 +79,91 @@
...
-        "string"
+        "object"
       ],
       "additionalProperties": false,
       "properties": {
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
```
Comment (koszti) [inline tap_twilio/schemas/cumulative_statistics.json:148]: is this OK with integer? Average usually can be floating point number

File: tap_twilio/schemas/cumulative_statistics.json [hunk 2]
```diff
@@ -79,91 +79,91 @@
...
           ]
         },
         "min": {
           "type": [
             "null",
-            "number"
+            "integer"
```
Comment (koszti) [inline tap_twilio/schemas/cumulative_statistics.json:154]: is this OK as integer? Is the wait duration always full measurements? (like seconds?)

File: tap_twilio/schemas/cumulative_statistics.json [hunk 3]
```diff
@@ -79,91 +79,91 @@
...
-        "string"
+        "object"
       ],
       "additionalProperties": false,
       "properties": {
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
```
Comment (koszti) [inline tap_twilio/schemas/cumulative_statistics.json:115]: is this OK as integer? Average usually can be floating point number

File: tap_twilio/schemas/cumulative_statistics.json [hunk 4]
```diff
@@ -79,91 +79,91 @@
...
```
Comment (koszti) [inline tap_twilio/schemas/cumulative_statistics.json:82]: is this OK as integer? Average usually can be floating point number

File: tap_twilio/schemas/cumulative_statistics.json (hunk 1 above)
Comment (tibortezsla) [reply tap_twilio/schemas/cumulative_statistics.json:148]: Their API doc does not define the type of the value, but it does for some which are not embedded into other resources, e.g. `avg_task_acceptance_time`, which is also a whole number.
I did some sample queries to their API, and they do return only whole numbers for these resources.

File: tap_twilio/schemas/cumulative_statistics.json (hunk 2 above)
Comment (tibortezsla) [reply tap_twilio/schemas/cumulative_statistics.json:154]: Yes, they always returned a whole number via their API (in their docs, they do not say anything specific about this)

File: tap_twilio/schemas/cumulative_statistics.json (hunk 3 above)
Comment (tibortezsla) [reply tap_twilio/schemas/cumulative_statistics.json:115]: Their API doc does not define the type of the value, but it does for some which are not embedded into other resources, e.g. `avg_task_acceptance_time`, which is also a whole number.
I did some sample queries to their API, and they do return only whole numbers for these resources.

File: tap_twilio/schemas/cumulative_statistics.json (hunk 4 above)
Comment (tibortezsla) [reply tap_twilio/schemas/cumulative_statistics.json:82]: Their API doc does not define the type of the value, but it does for some which are not embedded into other resources, e.g. `avg_task_acceptance_time`, which is also a whole number.
I did some sample queries to their API, and they do return only whole numbers for these resources.

File: tap_twilio/schemas/cumulative_statistics.json (hunk 3 above)
Comment (koszti) [reply tap_twilio/schemas/cumulative_statistics.json:115]: OK, thanks. If we'll receive a float avg number by any reason, the tap will fail on prod.
We just need to keep this in mind.

**Review by koszti — APPROVED** [review]

**Pull request merged**